### PR TITLE
fix(volar): prevent LS crash

### LINKED ISSUE #119: Suggections(vue): suggestions in vue projects broke in latest release
STR:
1. Vue project with options API (didn't test on composition api syntax), e.g. [this simple one](https://github.com/maIIady/pinia-test)
2.  Vue support is enabled: `"tsEssentialPlugins.enableVueSupport": true`
3.  Volar config path is overriden: `"volar.vueserver.configFilePath": "<PATH_TO_VSCODE>\\.vscode\\extensions\\zardoy.ts-essential-plugins-0.0.53\\volarConfig.js",`
4. Trigger suggestions in this line of mentioned project https://github.com/maIIady/pinia-test/blob/28a1ddcd5d366d4255ca88e43a94ccd31001baaf/src/App.vue#L39

AR:
image

P.S. I'm gonna eventually investigate it myself but I don't have time rn. 
@zardoy It definitely broke in one of the commits after this one - 56978448b4358f0d046aacac30ea8bc0ffbc4863 (everything was good when I developed it). Maybe you have an idea/quick suggestion where to look?

### PR DESCRIPTION
**Pull request by Ilanaya**
Closes #119

### CODE REVIEW

File: typescript/src/adjustAutoImports.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export function parseIgnoreSpec(spec: string): ParsedIgnoreSetting {
 }
 
 export const getIgnoreAutoImportSetting = (c: GetConfig) => {
-    return c('suggestions.ignoreAutoImports').map(spec => parseIgnoreSpec(spec))
+    return c('suggestions.ignoreAutoImports')?.map(spec => parseIgnoreSpec(spec))
```
Comment (maIIady) [inline typescript/src/adjustAutoImports.ts:69]: I had `reading properties of undefined` when started Volar debugger with `--disable-extensions`. I think these changes may make sense.

Comment (zardoy) [inline typescript/src/adjustAutoImports.ts:69]: Why is this needed? You should not get crash as all config should be passed from volarConfig. Maybe you had this extension disabled?

**Review by zardoy — CHANGES_REQUESTED** [review]

File: typescript/src/adjustAutoImports.ts (hunk 1 above)
Comment (zardoy) [reply typescript/src/adjustAutoImports.ts]: I would investigate this. https://github.com/zardoy/typescript-vscode-plugins/blob/develop/typescript/src/volarConfig.ts you see that plugin activates only after full configuration is received.
Btw could you also remove L26-28 in this or in this file?

Comment (zardoy) [reply typescript/src/adjustAutoImports.ts]: @maIIady I'm from mobile cannot do it now. Can you do it? This useless log really annoys me

Comment (maIIady) [reply typescript/src/adjustAutoImports.ts]: A moment

**Review by zardoy — CHANGES_REQUESTED** [review]
Can we regenerate pnpm lockfile? Cannot merge because of failing CI checks

**Review by zardoy — APPROVED** [review]
lgtm

**Comment by zardoy** [comment]
Thanks, but, just to note: You can always update / modify TypeScript version in  %localappdata%\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js

**Pull request merged**